test/server.jl

```diff
@@ -3,16 +3,20 @@ using HTTP.Test
 
 port = rand(8000:8999)
 
-function testget(url)
-    mktempdir() do d
-        cd(d) do
-            cmd = `"curl -v -s $url > tmpout 2>&1"`
-            cmd = `bash -c $cmd`
-            #println(cmd)
-            run(cmd)
-            return String(read(joinpath(d, "tmpout")))
-        end
+function testget(url,n=1)
+    p = Pipe()
+    if Sys.is_windows()
+        null = "NUL"
+    else
+        null = "/dev/null"
     end
+    cmd = `curl $(repeat(["-s","-o",null],outer=n)) -w '%{http_code}' $(repeat([url],outer=n))`
+    println(cmd)
+
+    run(pipeline(cmd,stdout=p))
```

Comment:
You can do this with less code:
```julia
open(`curl ...`) do io
    println(readstring(io))
end
```

Reply:
However, I don't 100% trust pipelining the output of command to Julia. See https://github.com/JuliaLang/julia/issues/14747
That's why I was using bash to dump the output into a file. I trust bash, and I trust Julia to read a file :)

Reply:
![image](https://user-images.githubusercontent.com/8954730/35027667-88b68d10-fb84-11e7-8178-bd373916534e.png)

I'll be happy to get it to run locally on Windows first and then AppVeyor should be easy 😅

Reply:
No, windows is not my thing. I think at this point I it might be best to use `@async HTTP.request(...)` on windows and leave a FIXME.

Reply:
I want to see green on AppVeyor. I'm not giving up so easy 😂

Reply:
If we used `remote` then at lest the request would come from a seperate process....
```julia
using Distributed # on 0.7
addprocs(1)
remote(u->HTTP.request("GET", u))("http://httpbin.org")
```

Reply:
I'm just playing with this now... 
```julia
function testget(url, n)
    remote((url, n)-> begin
        r = []
        @sync for i in 1:n
            @async push!(r, HTTP.request("GET", url))
        end
        return r
    end)(url, n)
end
```

Reply:
Is there a way you can use `curl` on Linux for `testget` that does not require `bash`? I can use `curl` no problem. The problem is `bash`.

Reply:
In that case, just us `is_windows()` and whatever combination of `open`, `pipeline` works for you on windows.

Reply:
See https://github.com/JuliaWeb/HTTP.jl/commit/57192e6a6df54d6623ab47d24bd310fea450c710